.circleci/config.yml

```diff
@@ -68,8 +68,6 @@ aliases:
       - *save_build_cache
       - store_test_results:
           path: test/results
-      - store_artifacts:
-          path: test/results
```

Comment:
I think we still want this here, we just want the path to be stored as `build`, so we could download and inspect what was built/deployed.

Reply:
And to be clear, I just mean this one — the other two should stay removed like you did.

Reply:
Okay, updated.  It now stores the build folder.